app/forms/declaration/AdditionalFiscalReference.scala

```diff
@@ -37,12 +42,12 @@ object AdditionalFiscalReference extends DeclarationPage {
     "reference" -> text()
       .verifying("declaration.additionalFiscalReferences.reference.empty", _.trim.nonEmpty)
       .verifying("declaration.additionalFiscalReferences.reference.error", isAlphanumeric and noLongerThan(15))
-  )(AdditionalFiscalReference.apply)(AdditionalFiscalReference.unapply)
+  )(AdditionalFiscalReference.build)(AdditionalFiscalReference.unapply)
 
   def form(): Form[AdditionalFiscalReference] = Form(mapping)
 }
 
-case class AdditionalFiscalReferencesData(references: Seq[AdditionalFiscalReference]) {
+case class AdditionalFiscalReferencesData(isRequired: Option[YesNoAnswer], references: Seq[AdditionalFiscalReference]) {
```

Comment:
I couldn't find any usage of this field `isRequired`. Is it still necessary to have?